src/main/groovy/io/seqera/wave/configuration/BlobCacheConfig.groovy

```diff
@@ -62,6 +62,9 @@ class BlobCacheConfig {
     @Value('${wave.blobCache.requestsMemory}')
     String requestsMemory
 
+    @Nullable
+    @Value('${wave.blobCache.url-signature-duration:10}')
+    Long urlSignatureDuration
```

Comment:
What unit is this?

Reply:
minutes

Reply:
changed to duration for better understanding

Reply:
Let's use a `Duration` type to avoid ambiguities

Reply:
done

Reply:
There's a poblem here 

https://github.com/seqeralabs/wave/blob/2e3e847b9750ce7563190a09183d183387065096/src/main/groovy/io/seqera/wave/service/blob/impl/BlobCacheServiceImpl.groovy#L77-L83


The signed URL is cached in Redis which has a longer TTL, therefore it can be served  expired URLs

Reply:
so should i use statusDuration?https://github.com/seqeralabs/wave/blob/3bc0dd0ec0bea030c4c87b348a9bf8aed3243a19/src/main/groovy/io/seqera/wave/configuration/BuildConfig.groovy#L71

Reply:
but there is a limit of 7 days for signature duration

Reply:
I think it requires to generate a new signed URL even if it has been cached in Redis, and therefore move `blobDownloadUri` just before the `return` statement, but I don't know the impact in the current implementation

Reply:
done